### ISSUE TITLE
We cannot reproduce the same result in the paper, can you please release the checkpoint?

### ISSUE THREAD

**Issue by TyroneZ3** [aarbelle/GroundingBySeparation#1]
This is a really good paper with an excellent idea, but we cannot reproduce the result. We trained the model with 4 a100 gpus, the batch size is 4, learning rate is 0.0001. We figure it's the same setting described in the paper. We only get 67.58% accuracy in F30K when trained with coco dataset against 72.60% in the paper.
